content/content-bundles/resources/types.properties

```diff
@@ -153,7 +153,8 @@ instr.options		 = Select or unselect resources types to indicate whether resourc
 instr.props		 	= Change the resource's details and then choose 'Update' at the bottom. 
 instr.replace 		 = To replace the current version of this file with an updated version, select your updated file here.
 instr.require		 = Required items marked with {0}
-instr.text		 	= Type in the text and click 'Continue' at the bottom. 
+instr.text		 	= Type in the text and click 'Continue' at the bottom.
+instr.title			= Use letters, numbers and the '-' & '_' characters (with no spaces).
```

Comment:
Is that a wind up? Are you trying to say that Oxford don't use commas properly? Lol.